01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,187 @@ using InteractiveUtils
+Nowadays, there is a kind of linear thinking model in which it is taken for granted that the process of knowledge production has a very specific way of occurring: first, academic and theoretical discoveries are made. And only then, with all of that formalization achieve, one can start thinking of putting it into practical use.
+This type of thinking is counterproductive because, being so widespread, it causes many people to be more concerned with constantly acquiring theoretical knowledge, rather than taking action and immersing themselves in practice.
+
+By looking at the history of technology and innovation a repeating pattern can be seen: there is a constant differentiation between the people who do and the people who watch and then narrate. 
+"The History Written by the Losers" would [Nassim Nicholas Taleb](https://en.wikipedia.org/wiki/Nassim_Nicholas_Taleb) said, the people that are doing stuff don't have time for writing. 
+And it is not only that the non-practitioners are the ones who write about the findings of others, but as time goes by and because of this prevailing thought about the order of knowledge creation, society ends up being convinced that there was indeed an arduous intellectual and academic work first, and then came its implementation.
+That –apparently– common sense in which knowledge is built from a purely intellectual work that can be done in the armchair at home, and that only after acquiring this sacred theoretical knowledge it is possible to come up with technology or innovation, is the one we need to question.
+
+This confusion in the order in which technological advances occur is seen constantly and in the most varied areas of the history of innovation. 
+Take, for example, the development of the jet engine. It really had nothing to do with the discoveries of physicists researching, but to the cleverness and practical heuristics based on trial and error that engineers have developed, although in academic books it is stated the other way around. Or in a really practical field, finances. There, senior traders that have been deploying several heuristics to make their trades, build portfolios that are much more complex than the ones generated by the pricing formulas that academics invented based on the trial-and-error experiences of people who are immersed in the market. 
+And this happened because, in the process of trying to generalize this heuristics into formal equations, the academics are constantly introducing fragilities. 
+That is, in the process of finding the laws that rules those dynamic systems, lot of cases are left behind, thing that does not happen to experienced traders. 
+And the worst thing? After doing so, finance PhDs can't understand how traders can correctly assing prices to financial derivatives without knowing several theorems... 
+So the wheel is broken.
```

Comment:
In this book we will try to take you, the reader, through a journey that is more similar to the real way in which knowledge is built: an iterative, hands-on process of problem-solving that gradually builds intuitions about how things work and why, that we can later formalize.

Reply:
I think the chapter could end here, for now. Please divide it into two PRs and we'll work on the rest later.